modules/logging-rn-5.6.12.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+// logging-5-6-release-notes.adoc
+:_content-type: REFERENCE
+[id="cluster-logging-release-notes-5-6-12_{context}"]
+= Logging 5.6.12
+This release includes link:https://access.redhat.com/errata/RHSA-2023:5541[OpenShift Logging Bug Fix Release 5.6.12].
+
+[id="openshift-logging-5-6-12-bug-fixes_{context}"]
+== Bug fixes
+* Before this update, deploying a LokiStack on IPv6-only or dual-stack {product-title} clusters caused the LokiStack memberlist registration to fail. As a result, the distributor pods went into a crash loop. With this update, an administrator can enable IPv6 by setting the `lokistack.spec.hashRing.memberlist.enableIPv6:` value to `true`, which resolves the issue. (link:https://issues.redhat.com/browse/LOG-4570[LOG-4570])
```

Comment:
We need the same known issue hint as here: https://github.com/openshift/openshift-docs/pull/66871/files#diff-e81d83066b3b88da7b4f6aa7fb7c6457b2a30c18c0a02c54db77918d7687728fR56 because IPv6 LokiStack with the ruler enabled can't process alert queries.

Reply:
Added this as a one liner after the release note text.